src/modules/did/did.module.ts

```diff
@@ -40,8 +41,9 @@ const RegistrySettingsProvider = {
     RegistrySettingsProvider,
     {
       provide: DidStore,
-      useFactory: (ipfsConfig) => {
-        return new DidStore(ipfsConfig);
+      useFactory: ({ protocol, host, port, headers }: IpfsConfig) => {
+        const url = `${protocol}://${host}:${port}`;
+        return new DidStore(url, headers);
```

Comment:
@JGiter why is this better than passing in the config object directly?

Reply:
At the time, when PR was created current version of did-ipfs-store (0.7.1-alpha.712) was ipfs cluster based, where first constructor param was string, but later in 0.7.1-alpha.795 we returned to ipfs gateway. I'll revert this change. For some reason version change isn't shown in this PR